src/app/App.js

```diff
@@ -61,6 +61,14 @@ class App extends EventEmitter {
 		this.captureScrollPositionFromScrollEvent = true;
 
 		/**
+		 * Holds the value of the current browser path.
+		 * @type {!string}
+		 * @default the current browser path.
+		 * @protected
+		 */
+		this.currentBrowserPath = window.location.pathname;
```

Comment:
I see. Yeah, that would make it much more clear. It could be previousBrowserPath or browserPathBeforeNavigate...